package.json

```
@@ -7,7 +7,7 @@
     "require-all": "^3.0.0"
   },
   "name": "shotgun-nodejs",
-  "version": "4.2.0",
+  "version": "4.2.5",
```

It's a very marketing thing to increment versions like this. In technical space, we should always do it by 1.
```suggestion
  "version": "4.2.1",
```